### PR TITLE
Fixed issue #18984: CSRF in Question Themes function

### PR DESCRIPTION
**Pull request by gabrieljenik**
Function toggleVisibility is now requested via POST.

### CODE REVIEW

File: application/views/themeOptions/installedthemelist.php [hunk 1]
```diff
@@ -96,12 +96,8 @@ function(id, data){
...
                                         $.ajax({
                                             url : $url,
-                                            type : "GET",
-                                            dataType : "html",
-                                
-                                            // html contains the buttons
-                                            success : function(html, statut){
-                                            }
+                                            method: "POST",
+                                            data: "",
```
Comment (Shnoulle) [inline application/views/themeOptions/installedthemelist.php:100]: `{}` ? maybe

**Review by Shnoulle — COMMENTED** [review]
Seems OK, why draft ?

**Comment by gabrieljenik** [comment]
All good. Wanted to make sure the CSRF was sent.

image

**Comment by Shnoulle** [comment]
> All good. Wanted to make sure the CSRF was sent.
> 

Since you ask it with `$this->requirePostRequest();` : it was sent, no ?

**Review by Shnoulle — APPROVED** [review]

**Pull request merged**